if-cycles/com/statements/examples/LambdaExample1.java

```diff
@@ -0,0 +1,86 @@
+package com.statements.examples;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Stream;
+
+class Cat {
+	private String name;
+	private String caste;
+	private String nativeLand;
+	private int age;
+
+	public Cat() {
+	}
+
+	public Cat(String _name, String _caste, String _nativeLand, int _age) {
+		this.name = _name;
+		this.caste = _caste;
+		this.nativeLand = _nativeLand;
+		this.age = _age;
+	}
+
+	public int getAge() {
+		return age;
+	}
+
+	public List<Cat> AddCat() {
```

Comment:
I think it's a bad idea to use this in your entity, entity usually have only variables and set, get methods, I think that better if it's will be in another class that will contain logic for Cat

Use
public void setCats(List<Cat> cats){}
public void List<Cat> getCats(){}
This applies also other methods in this class